src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -307,13 +307,10 @@ public boolean setSpawnLocation(int x, int y, int z)
 	@Override
 	public @NotNull List<Entity> getEntities()
 	{
-		// MockBukkit.assertMocking();
-		List<Entity> entities = new ArrayList<>();
-
-		Collection<? extends EntityMock> serverEntities = MockBukkit.getMock().getEntities();
-		serverEntities.stream().filter(entity -> entity.getWorld() == this)
-		.collect(Collectors.toCollection(() -> entities));
-		return entities;
+		return server.getEntities().stream()
+				.filter(entity -> entity.getWorld() == this)
+//				.filter(EntityMock::isValid)
```

Comment:
```suggestion
```

Reply:
That should be there, but it's only implemented in #332 so I left it commented out as a sort of reminder. Would you prefer if I left a todo?